astropy/io/votable/tests/table_test.py

```diff
@@ -175,6 +180,93 @@ def test_write_with_format():
 def test_empty_table():
     votable = parse(
         get_pkg_data_filename('data/empty_table.xml'),
-        pedantic=False)
+        verify='warn')
     table = votable.get_first_table()
     astropy_table = table.to_table()  # noqa
+
+
+class TestVerifyOptions:
+
+    # Start off by checking the default (ignore)
+
+    def test_default(self):
+        with catch_warnings(VOWarning) as w:
+            parse(get_pkg_data_filename('data/gemini.xml'))
+        assert len(w) == 0
+
+    # Then try the various explicit options
+
+    def test_verify_ignore(self):
+        with catch_warnings(VOWarning) as w:
+            parse(get_pkg_data_filename('data/gemini.xml'), verify='ignore')
+        assert len(w) == 0
+
+    def test_verify_warn(self):
+        with catch_warnings(VOWarning) as w:
+            parse(get_pkg_data_filename('data/gemini.xml'), verify='warn')
+        assert len(w) == 25
+
+    def test_verify_exception(self):
+        with pytest.raises(VOWarning):
+            parse(get_pkg_data_filename('data/gemini.xml'), verify='exception')
+
+    # Make sure the pedantic option still works for now (pending deprecation)
+
+    def test_pedantic_false(self):
+        with catch_warnings(VOWarning) as w:
```

Comment:
To check for it below, you first need to catch it. (Gotta catch them all... Pokemon!)
```suggestion
        with catch_warnings(VOWarning, AstropyDeprecationWarning) as w:
```